app/service/views/rest.py

```diff
@@ -67,3 +72,49 @@ def get_service(service_id=None):
         return jsonify(result="error", message="Service not found"), 404
     data, errors = services_schema.dump(services) if isinstance(services, list) else service_schema.dump(services)
     return jsonify(data=data)
+
+
+# TODO auth to be added
+@service.route('/<int:service_id>/token', methods=['POST'])
+def create_token(service_id=None):
```

Comment:
Is a token created through a separate api call, or automatically when a service is created?